esmvaltool/references/mckee93.bibtex

```diff
@@ -0,0 +1,3 @@
+@article{mckee93,
```

Comment:
Please split the fields.
We could rename this `mckee93proc`.

Reply:
done.